DigitalLearningSolutions.Data.Tests/TestHelpers/UserTestHelper.cs

```diff
@@ -156,5 +177,25 @@ FROM Candidates
 
             return users.Single();
         }
+
+        public static async Task<AdminUser> GetAdminUserByEmailAddressAsync(

```

Comment:
We don't need this method.

We should always prefer to use the existing method in the data service that serves the same purpose rather than defining new queries for tests since they should already have unit tests so we know they work correctly. The above delegate users one has no equivalent data service method.